sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CreateTest.java

```diff
@@ -224,7 +224,7 @@ public UnserializableRecord decode(
   }
 
   @Test
-  @Category(ValidatesRunner.class)
+  @Category(NeedsRunner.class)
```

Comment:
How many runners are currently overriding Create as a primitive? If they aren't, then done! If they are, then (and this is a new thought that I thought of after my prior comments) another place for the tests to move is into ITs for just that runner, via some organization of modules. This is after-this-PR suggestion. I'm OK with this as-is unless runners are all turning it back into a primitive.

Reply:
Ack, I'll leave as-is.